### PR TITLE
Add the ability to show/hide channels in time series

### PR DESCRIPTION
**Pull request by daniellasry**
- Show/hide channels
- rescale channelbar accordingly
- show/hide rows in HW SettingsController

### CODE REVIEW

File: OpenBCI_GUI/W_BandPower.pde [hunk 1]
```diff
@@ -117,7 +117,6 @@ class W_BandPower extends Widget {
 
         popStyle();
         bpChanSelect.draw();
-        pushStyle();
```
Comment (retiutut) [inline OpenBCI_GUI/W_BandPower.pde:120]: 👍

File: OpenBCI_GUI/W_TimeSeries.pde [hunk 2]
```diff
@@ -101,7 +104,7 @@ class W_timeSeries extends Widget {
         }
 
         if(currentBoard instanceof ADS1299SettingsBoard) {
-            hardwareSettingsButton = new Button_obci((int)(x + 3), (int)(y + navHeight + 3), 120, navHeight - 6, "Hardware Settings", 12);
+            hardwareSettingsButton = new Button_obci((int)(x + 90), (int)(y + navHeight + 3), 120, navHeight - 6, "Hardware Settings", 12);
```
Comment (retiutut) [inline OpenBCI_GUI/W_TimeSeries.pde:107]: Heading to bed, but this was the last thing to check. +90px to the right sounds like enough space for channelSelect dropdown.

File: OpenBCI_GUI/W_TimeSeries.pde [hunk 3]
```diff
@@ -136,6 +139,22 @@ class W_timeSeries extends Widget {
...
+            channelBarHeight = int(ts_h/numChannelBars);
+
+            for(int i = 0; i < tsChanSelect.activeChan.size(); i++) {
+                int activeChan = tsChanSelect.activeChan.get(i);
+                int channelBarY = int(ts_y) + i*(channelBarHeight); //iterate through bar locations
+                channelBars[activeChan].resize(int(ts_x), channelBarY, int(ts_w), channelBarHeight); //bar x, bar y, bar w, bar h
+            }
+
+            if (currentBoard instanceof ADS1299SettingsBoard) {
+                hardwareSettingsButton.setPos((int)(x0 + 90), (int)(y0 + navHeight + 3));
```
Comment (retiutut) [inline OpenBCI_GUI/W_TimeSeries.pde:154]: Can we do `x0 + 80`?

Screen Shot 2020-06-25 at 12 52 52 AM

Screen Shot 2020-06-25 at 1 00 17 AM

File: OpenBCI_GUI/W_TimeSeries.pde (hunk 2 above)
Comment (retiutut) [reply OpenBCI_GUI/W_TimeSeries.pde:107]: And here `x0 + 80` also.

**Comment by daniellasry** [comment]
@retiutut added flex and changed to +80

**Comment by retiutut** [comment]
💯

**Pull request merged**